src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
```

Comment:
<img width="442" alt="Screenshot 2023-10-09 at 16 10 16" src="https://github.com/mate-academy/js_task-transportation-on-vacation/assets/135232001/939f0de0-9c9a-4083-92f0-fa9daec2d8d9">
Consider following the checklist to write code according to the best practices